app/scripts/views/permissions.js

```diff
@@ -0,0 +1,135 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+'use strict';
+
+define([
+  'cocktail',
+  'underscore',
+  'views/form',
+  'views/base',
+  'stache!templates/permissions',
+  'lib/promise',
+  'views/mixins/back-mixin',
+  'views/mixins/service-mixin'
+],
+function (Cocktail, _, FormView, BaseView, Template, p,
+    BackMixin, ServiceMixin) {
+
+  var View = FormView.extend({
+    template: Template,
+    className: 'permissions',
+
+    initialize: function (options) {
+      // Account data is passed in from sign up and sign in flows.
+      var data = this.ephemeralData();
+      this._account = data && this.user.initAccount(data.account);
+
+      this.type = options.type;
+    },
+
+    getAccount: function () {
+      return this._account;
+    },
+
+    context: function () {
+      var account = this.getAccount();
+      return {
+        email: account.get('email'),
+        serviceName: this.relier.get('serviceName'),
+        termsUri: this.relier.get('termsUri'),
+        privacyUri: this.relier.get('privacyUri'),
+        serviceUri: this.relier.get('serviceUri')
+      };
+    },
+
+    events: {
+      // validateAndSubmit is used to prevent multiple concurrent submissions.
+      'click #proceed': BaseView.preventDefaultThen('validateAndSubmit')
```

Comment:
This declaration is not needed since this is based off of FormView, which already has a form submit handler.